### DISCUSSION TITLE
IPI regulation

### DISCUSSION THREAD

**Discussion by EvijaMozga** [OP-TED/eForms-SDK#1049] [Q&A] [answered]
Dear TED team,

According to eForms Guideline on the IPI: [Finalise IPI guide (#16) · Issues · eProcurement Group / eForms Group / Change Requests · GitLab](https://code.europa.eu/eproc/eforms/crs/-/issues/16), if it is indicated that an IPI measure is applicable to this EU public procurement procedure, more information needs to be provided under the business group Information on IPI Measures (BG-681). It is necessary to provide the Number of tender applications that fall under this IPI Measure (BT-686) for each Specific IPI Measure (BT-685). As it is possible to have different types of Specific IPI Measure (BT-685), it is possible to provide multiple measures and to mention for each of them the Number of tender applications that fall under this IPI Measure (BT-686).

The problem is that in the table of the 2nd amendments of e-forms (in the attachment), IPI activities should be indicated against the tenders (BG-320), not against the results (BG-137). The table suggests that IPI activities should be indicated for each tender, but this does not align with the previously mentioned business process description (which states that the applicable IPI activities should be indicated alongside the corresponding number of tenders).

Currently, SDK version 12 is available. However, the fields for IPI regulation are planned only for SDK version 13, which is not yet available. Therefore, it’s unclear how the IPI measures will be recorded in the scheme.
 Could you clarify how to correctly indicate IPI information according to the EU OJ scheme?

Thank you in advance!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau
[eforms_table2_2023-2884.xlsx](https://github.com/user-attachments/files/17504585/eforms_table2_2023-2884.xlsx)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Issues are exclusively used for SDK specific issues, so this has been changed to a Discussion/Q&A.
Regarding the issue raised for the association with LotTender vs LotResult, you should get in touch directly with DG GROW. As you reminded, aggregated information is expected which for a given Tender wouldn't be suitable while for a given LotResult makes sense.
There isn't currently any Implementing Act and no IPI information may currently be reported.
When the documentation of SDK 1.13 will get published, you'll be able to find an illustrative example I've copied below. Note that the example is only there for illustration purpose and not for consistency. Focus was to show the implementation for the various BTs. Schema is already available in the SDK 1.13 RC1

```
        <efac:StrategicProcurement>
            <efac:StrategicProcurementInformation>
                <!-- Information on IPI Measures (BG-681) -->
                <efac:AppliedMeasure>
                    <!-- Specific IPI Measure (BT-685) -->
                    <efbc:MeasureCode listName="international-procurement-instrument-measure">empty</efbc:MeasureCode>
                    <efac:MeasureException>
                        <!-- exception to the application of the IPI Measure (BT-687) -->
                        <efbc:ExceptionCode listName="international-procurement-instrument-application">pub-int</efbc:ExceptionCode>
                        <!-- Overriding reasons relating to the public interest (BT-688) -->
                        <efbc:ExceptionDescription languageID="EN">because ...</efbc:ExceptionDescription>
                    </efac:MeasureException>
                    <!-- Number of tender applications that fall under this IPI Measure (BT-686) -->
                    <efac:MeasureStatistics>
                        <efbc:StatisticsNumeric>3</efbc:StatisticsNumeric>
                    </efac:MeasureStatistics>
                </efac:AppliedMeasure>
            </efac:StrategicProcurementInformation>
        </efac:StrategicProcurement>
```
